includes/data-port/models/class-sensei-import-course-model.php

```diff
@@ -279,4 +285,142 @@ private function delete_course_terms( $course_id, $taxonomy ) {
 			delete_post_meta( $course_id, '_module_order' );
 		}
 	}
+
+	/**
+	 * Migrates the imported post content to use the ids of the newly created lessons and modules.
+	 *
+	 * @param string $post_content The post content.
+	 *
+	 * @return string The migrated post content.
+	 */
+	private function migrate_post_contnet( $post_content ) {
```

Comment:
Typo here in the method name.

Reply:
Fixed in a890740ba26ded83631f927dc4a831f99c4553f9